src/test/java/com/at/globalclasses/domain/UserRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.at.globalclasses.domain;
+
+
```

Comment:
Let's add lombok dependency so we can avoid boilerplate code

Reply:
I've sent a commit with this change